sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -353,6 +356,42 @@ object Std {
       }
       res
     },
+    builtin("member", "arr", "x"){ (ev, fs, arr: Val, x: Val) =>
+      val res = arr match {
+        case str: Val.Str =>
+          val secondArg = x match {
+            case Val.Str(value) => value
+            case n => Materializer(x)(ev).toString //The behavior of 'member' when first arg is string and second arg is not string is undefined - we do our best here
```

Comment:
It seems that `std.member` simply raises an error if the second argument `x` is not a string, at least when i tested it on jsonnet.org. Let's follow suite

Reply:
Agreed, will fix.